### PR TITLE
Update to 1.21.2/3

### PR DESCRIPTION
**Pull request by DrexHD**

### CODE REVIEW

**Comment by viperxx** [comment]
Just built and installed on my test server, working great so far, thank you for updating!

**Comment by P5ina** [comment]
Is it ready to use in production?

**Comment by DrexHD** [comment]
Should be usable. Waiting for @PotatoPresident to review.

File: src/main/kotlin/com/github/quiltservertools/ledger/actions/ItemDropActionType.kt [hunk 1]
```diff
@@ -23,7 +22,7 @@ open class ItemDropActionType : AbstractActionType() {
 
     override fun getTranslationType(): String {
         val item = Registries.ITEM.get(objectIdentifier)
-        return if (item is BlockItem && item !is AliasedBlockItem) {
+        return if (item is BlockItem) {
```
Comment (PotatoPresident) [inline src/main/kotlin/com/github/quiltservertools/ledger/actions/ItemDropActionType.kt:25]: This breaks with items like wheat seeds, which are block items but only have item translations

**Review by PotatoPresident — CHANGES_REQUESTED** [review]

**Review by PotatoPresident — DISMISSED** [review]

**Review by PotatoPresident — APPROVED** [review]

**Pull request merged**